core/arch/x86/instr_create.h

```diff
@@ -2233,6 +2236,29 @@
     instr_create_1dst_2src((dc), OP_kshiftrd, (d), (s1), (s2))
 /* @} */ /* end doxygen group */
 
+/** @name 1 destination, 1 mask and 1 non-immediate source */
```

Comment:
nit: DR style is a comma before "and" so s/mask/mask,/  Ditto below.

Reply:
Done.